src/main.ts

```diff
@@ -111,6 +114,12 @@ async function run(): Promise<void> {
     }
 
     // Validation
+    if (httpsTriggerSecurityLevel && eventTriggerType) {
```

Comment:
Since `https_trigger_security_level` has a default wouldn't `httpsTriggerSecurityLevel` be truthy by default?

Reply:
Good point. It turns out that it's easier to fix this if we set the default value to "SECURITY_LEVEL_UNSPECIFIED" (same as gcloud).